p2p/p2p.go

```diff
@@ -11,7 +11,7 @@ import (
 // All implementations that will be called from Messenger implementation will need to satisfy this interface
 // If the function returns a non nil value, the received message will not be propagated to its connected peers
 type MessageProcessor interface {
-	ProcessReceivedMessage(message MessageP2P, broadcastHandler func(buffToSend []byte)) error
+	ProcessReceivedMessage(message MessageP2P, fromConnectedPeer PeerID, broadcastHandler func(buffToSend []byte)) error
```

Comment:
the broadcasthandler is used just in one call and in all other calls it is ignored. Maybe remove the broadcasthandler or create a new interface for the processor that uses 3 params on the processReceivedMessage.

Reply:
Removed completely as agreed. MultiDataInterceptor no longer cleans received data for other peers.

Reply:
ok!